app/Http/Resources/RequestCanvassSummaryDetailedResource.php

```
@@ -16,15 +16,29 @@
             'terms_of_payment' => $this->terms_of_payment,
             'availability' => $this->availability,
             'delivery_terms' => $this->delivery_terms,
             'remarks' => $this->remarks,
-            'price_quotation_id' => $this->priceQuotation->id,
-            'supplier' => new RequestSupplierDetailedResource($this->priceQuotation->supplier),
-            'items' => CanvassSummaryItemDetailedResource::collection($this->items),
+            'price_quotation_id' => $this->whenLoaded('priceQuotation', fn () => $this->priceQuotation->id),
+            'suppliers' => PriceQuotationForCanvassResource::collection($this->ordered_suppliers),
+            'items' => $this->whenLoaded(
+                'items',
+                fn () => CanvassSummaryItemDetailedResource::collection($this->items)
```

PHPMD (2.15.0)

Avoid using static access to class 'App\Http\Resources\CanvassSummaryItemDetailedResource' in method 'toArray'. (Clean Code Rules)

(StaticAccess)